### ISSUE TITLE
利用BMN训练足球数据集时报错

### ISSUE THREAD

**Issue by EnzoLiang** [PaddlePaddle/PaddleVideo#205]
**Describe the bug**
按照FootballAction文档执行到step6 后得到了frames的feature文件.npy，其中shape为（200，2048）。然后进行BMN模型训练，发现报以下错误：

![微信截图_20210924112618](https://user-images.githubusercontent.com/27463147/134624901-e44d0066-17df-4d6b-b079-8fa70c3dcef7.png)

**To Reproduce**
1. 运行环境为ubuntu16.04，paddle-gpu==2.1.2, paddlevideo==0.0.1
2. BMN配置文件为：
MODEL:                                              #MODEL field
  framework: "BMNLocalizer"
  backbone:
    name: "BMN"
    # pretrained: "/mnt/7T/gj/liangyzh/projects/FootballAction/pretrain/ResNet50_vd_ssld_v2_pretrained.pdparams"
    # feat_dim: 2688
    tscale: 200
    dscale: 200
    prop_boundary_ratio: 0.5
    num_sample: 32
    num_sample_perbin: 3
  loss:
    name: "BMNLoss"
    tscale: 200
    dscale: 200

DATASET:                                            #DATASET field
  batch_size: 4     # 4                                     #single card bacth size
  test_batch_size: 1
  num_workers: 8
  train:
    format: "BMNDataset"
    file_path: "/mnt/80.241-iscsi-1/025-liangyzh/dev/projects/FootballAction/datasets/EuroCup2016/input_for_bmn/label.json"
    subset: "train"
  valid:
    format: "BMNDataset"
    file_path: "/mnt/80.241-iscsi-1/025-liangyzh/dev/projects/FootballAction/datasets/EuroCup2016/input_for_bmn/label.json"
    subset: "validation"
  test:
    format: "BMNDataset"
    test_mode: True
    file_path: "/mnt/80.241-iscsi-1/025-liangyzh/dev/projects/FootballAction/datasets/EuroCup2016/input_for_bmn/label.json"
    subset: "validation"

PIPELINE:                                           #PIPELINE field
  train:                                            #Mandotary, indicate the pipeline to deal with the training data
    load_feat:
      name: "LoadFeat"
      feat_path: "/mnt/80.241-iscsi-1/025-liangyzh/dev/projects/FootballAction/datasets/EuroCup2016/input_for_bmn/feature"
    transform:                                      #Mandotary, image transfrom operator
      - GetMatchMap:
          tscale: 200
      - GetVideoLabel:
          tscale: 200
          dscale: 200

  valid:                                            #Mandotary, indicate the pipeline to deal with the training data
    load_feat:
      name: "LoadFeat"
      feat_path: "/mnt/80.241-iscsi-1/025-liangyzh/dev/projects/FootballAction/datasets/EuroCup2016/input_for_bmn/feature"
    transform:                                      #Mandotary, image transfrom operator
      - GetMatchMap:
          tscale: 200
      - GetVideoLabel:
          tscale: 200
          dscale: 200

  test:                                             #Mandatory, indicate the pipeline to deal with the validing data
    load_feat:
      name: "LoadFeat"
      feat_path: "/mnt/80.241-iscsi-1/025-liangyzh/dev/projects/FootballAction/datasets/EuroCup2016/input_for_bmn/feature"
    transform:                                      #Mandotary, image transfrom operator
      - GetMatchMap:
          tscale: 200
      - GetVideoLabel:
          tscale: 200
          dscale: 200

OPTIMIZER:                                          #OPTIMIZER field
  name: 'Adam'
  learning_rate:
    iter_step: True
    name: 'CustomPiecewiseDecay'
    boundaries: [4200]
    values: [0.001, 0.0001]
  weight_decay:
    name: 'L2'
    value: 1e-4

METRIC:
  name: 'BMNMetric'
  tscale: 200
  dscale: 200
  # file_path: "data/dataset/bmn_data/activitynet_1.3_annotations.json"
  # ground_truth_filename: "data/dataset/bmn_data/activity_net_1_3_new.json"
  # subset: "validation"
  # output_path: "data/bmn/BMN_Test_output"
  # result_path: "data/bmn/BMN_Test_results"

model_name: BMN
epochs: 20                                          #Mandatory, total epoch
log_level: "INFO"
resume_from: ""                                     #checkpoint path.

3. debug时发现经dataloader输入BMN的数据shape不一致，如下图：
![微信图片_20210924135718](https://user-images.githubusercontent.com/27463147/134625445-9e0fcc5a-a2c6-4bd9-93ea-db40470b831e.png)

**Comment by wgh1989** [comment]
是BMN的输入参数问题
![157885b3e6ff7c611cd0f118fcd8336a](https://user-images.githubusercontent.com/35423940/134628739-140e1e86-2837-4488-bcd8-186b70149cf0.png)
这里写死了，抱歉，我们会尽快修复
如果等不了，可以先用这个版本，适合football-action所有步骤
https://bj.bcebos.com/v1/acg-algo/models/action-football/PaddleVideo-release-2.1-football.tar
